src/codeManager.ts

```diff
@@ -158,7 +158,15 @@ export class CodeManager {
                 fileType = '.' + this._languageId;
             }
         }
-        let tmpFileName = 'temp-' + this.rndName() + fileType;

```

Comment:
Let's keep it simple. Just remove '-' is OK: `let tmpFileName = 'temp' + this.rndName() + fileType;`